### DISCUSSION TITLE
Filter by repeated fields /some_endpoint?sth=1&sth=2&sth=3

### DISCUSSION THREAD

**Discussion by Przemek625** [carltongibson/django-filter#1411] [Q&A]
How to write a filter that will filter by multiple repeated params. For example:

`/some_endpoint?sth=1&sth=2&sth=3`

Lets assume that sth is `IntegerField`. It would be good if it could also valudate if the gived param is also Integer in this case. 

**I went through the documentation and I find it extremely difficult to do sth as simple as that.** 

I found example that allows to filter by values seperated with `,`. I mean this code:

```
class NumberInFilter(df.BaseInFilter, df.NumberFilter):
    pass
```
But this does not meet my requirements and I am not interested in this class. 

For examle in django rest framework it is very easy to achieve this. It is as simple as that :
```

class SomeSerializer(serializers.Serializer):
    sth = serializers.ListField(
        required=False,
        child=serializers.CharField(),
    )
```

**Comment by carltongibson** [comment]
See the [docs for `MultipleChoiceFilter`](https://django-filter.readthedocs.io/en/stable/ref/filters.html#multiplechoicefilter) - this is exactly what it does.
